debian/patches/fix-hardening

```diff
@@ -0,0 +1,40 @@
+From fcb8b6d6bf2e5325fa03adc11e4cc4741540ce49 Mon Sep 17 00:00:00 2001
+From: Frederic Bonnard <frediz@linux.vnet.ibm.com>
+Date: Fri, 29 May 2015 15:50:20 +0200
+Subject: [PATCH] Honor LDFLAGS and CPPFLAGS
```

Comment:
This patch has already been integrated into the repository and won't be necessary.